app/build.gradle

```diff
@@ -0,0 +1,33 @@
+apply plugin: 'com.android.application'
+
+apply plugin: 'kotlin-android'
+
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion 27
+    defaultConfig {
+        applicationId "com.example.weathercheckerapp"
+        minSdkVersion 21
+        targetSdkVersion 28
+        versionCode 1
+        versionName "1.0"
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation"org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation 'com.android.support:appcompat-v7:27.1.1'
```

Comment:
Please use jetpack dependencies with the latest versions (as we use 3.2 gradle plugin which is not on stable yet, we can use here the not stable versions as well):

```
implementation "androidx.appcompat:appcompat:1.0.0-rc02"
implementation "androidx.constraintlayout:constraintlayout:2.0.0-alpha2"

androidTestImplementation "androidx.test:runner:2.0.0-rc01"
androidTestImplementation "androidx.test.espresso:espresso-core:3.1.0-alpha4"
```

Reply:
added "androidx.test:runner:1.1.0-alpha4" instead of "2.0.0-rc01"